modules/apps/style-book/style-book-api/src/main/java/com/liferay/style/book/importer/StyleBookEntryImporterEntry.java

```diff
@@ -10,9 +10,9 @@
 /**
  * @author Anderson Luiz
  */
-public class StyleBookEntryImporterImportResultEntry {
```

Comment:
@ethib137 asked me to use this pattern

Reply:
Aaah I understand why have "Result" in the name. Well it's the result data of the import...

So instead of `StyleBookEntryImporterEntry` maybe `StyleBookEntryImporterResult` ?

@ethib137

Reply:
This started as `StyleBookEntryZipProcessorImportResultEntry`. So `StyleBookEntryImporterImportResultEntry` is staying as close to that as possible. It's fine to change it a bit more and so in that case `StyleBookEntryImporterResult` probably makes the most sense. I'm fine with either of those.

Reply:
Cool, let's go with `StyleBookEntryImporterResult` then. Do you agree @algobob?